app/views/work_packages/bulk/destroy.html.erb

```diff
@@ -45,29 +45,41 @@ See doc/COPYRIGHT.rdoc for more details.
     </p>
 
     <%= fields_for :to_do do |f| %>
-      <div class="form--field">
+      <div class="form--field -trailing-label">
         <%= styled_label_tag :to_do_action_destroy, l(:text_destroy) %>
         <div class="form--field-container">
           <%= styled_radio_button_tag 'to_do[action]', 'destroy' %>
         </div>
       </div>
-      <div class="form--field">
+      <div class="form--field -trailing-label">
         <%=styled_label_tag 'to_do_action_nullify', l(:text_assign_to_project) %>
         <div class="form--field-container">
           <%= styled_radio_button_tag 'to_do[action]' , 'nullify' %>
         </div>
       </div>
-      <div class="form--field">
-        <%= styled_label_tag 'to_do_action_reassign', l(:text_reassign) %>
-        <div class="form--field-container">
-          <%= styled_radio_button_tag 'to_do[action]', 'reassign', :onclick => 'if(jQuery("#to_do_action_reassign").prop("checked")) { jQuery("#to_do_reassign_to_id").focus(); }' %>
-          <%= styled_label_tag 'to_do_reassign_to_id', l(:text_reassign_to) %>
```

Comment:
You should not simply delete that label, since it carries contextual information for impaired users.

At the very least you should add an invisible label again (`class="hidden-for-sighted"`), that way a screen reader will read it out loud, when focussing that label (e.g. "ID; edit text" vs "work package with hint: ID; edit text" for ChromeVox).